db_checker.py

```diff
@@ -342,22 +321,18 @@ def set_up(directories):
     args = parser.parse_args()
     xml_dir = args.output_dir[0]
 
-    # Load files
-    try:
-        set_up(args.input_dir)
-    except ValueError as err:
-        print(err.message)
-        sys.exit(False)
+    print "\n\n------ BEGINNING PV UNIT TESTS ------"
+    start = time.time()
 
-    # Load tests
-    units_suite = unittest.TestLoader().loadTestsFromTestCase(TestPVUnits)
+    loader = unittest.TestLoader()
+    suite = unittest.TestSuite()
 
-    print "\n\n------ BEGINNING PV UNIT TESTS ------"
+    for db in set_up(args.input_dir):
```

Comment:
This scanning step is by far the slowest operation. It is uncharacteristic of test scripts and impatient people (i.e. me) could interpret the lack of action as a hang. A line of output to keep the user informed would be helpful.